libnmstate/nm/ieee_802_1x.py

```diff
@@ -17,11 +17,11 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
-from libnmstate.error import NmstateLibnmError
 from libnmstate.schema import Ieee8021X
 
 from .common import NM
 from .common import GLib
+from .utils import get_secrets
```

Comment:
If we have file called `utils.py`, it will eventually grow bloated. Please use `secret.py` instead.